### PR TITLE
Add test info to README

### PR DESCRIPTION
**Pull request by superstar54**
Add a development section with test info into the README. This will be helpful for the new developer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,21 @@ to install the default SSSP version.
 List the installed pseudopotential families with the command `aiida-pseudo list`.
 You can then use the name of any family in the command line using the `-F` flag.
 
+## Development
+
+### Test
```
Comment (mbercx) [inline README.md:91]:
```suggestion
### Running tests
```

File: README.md [hunk 2]
```diff
@@ -86,6 +86,21 @@ to install the default SSSP version.
...
+```shell
```
Comment (mbercx) [inline README.md:92]: Maybe preface this with "To run the tests, simply clone and install the package locally with the `[tests]` optional dependencies:"

File: README.md [hunk 3]
```diff
@@ -86,6 +86,21 @@ to install the default SSSP version.
...
+git clone https://github.com/aiidateam/aiida-quantumespresso .
+cd aiida-quantumespresso
+pip install -e .[tests]  # install extra dependencies for test
+pytest tests/calculations/test_pw.py # run tests
```
Comment (mbercx) [inline README.md:96]: Maybe we want to show how to run all tests by default?

```suggestion
pytest # run tests
```

You can also add an example to show how to run a specific set of tests, but we should assume a developer understands the basics of `pytest`.

File: README.md [hunk 4]
```diff
@@ -86,6 +86,21 @@ to install the default SSSP version.
...
+```
+
+One can also use `tox` to run the test. `tox` allows setting the Python version for the test.
```
Comment (mbercx) [inline README.md:99]:
```suggestion
You can also use `tox` to run the test set. Here you can also use the `-e` option to specify the Python version for the test run:
```

**Review by mbercx — CHANGES_REQUESTED** [review]
Thanks @superstar54! I left some comments, but good to start adding some more instructions for developers.

I'm ok with adding these to the README, but maybe the documentation or wiki might be a better place? @sphuber what do you think?

**Comment by sphuber** [comment]
> I'm ok with adding these to the README, but maybe the documentation or wiki might be a better place? @sphuber what do you think?

I think for now the README is fine. If I visit other projects myself to develop, the first place I look is the README of the repo. I think the wiki is not an intuitive first place to look. I know we use it for AiiDA for certain things, but that is another story. So for me it is fine to add this now.

@superstar54 Maybe we can also add the following section
```
### Pre-commit
To contribute to this repository, please enable pre-commit so the code in commits are conform to the standards.
Simply install the repository with the `pre-commit` extra dependencies:
```shell
cd aiida-quantumespresso
pip install -e .[pre-commit]
pre-commit install
```
```

**Comment by mbercx** [comment]
@superstar54 and while you're at it, maybe fix the typo in the Acknowle_d_gements header :D

**Comment by superstar54** [comment]
@mbercx Thanks for the suggestions.
Thanks @sphuber, `pre-commit` section is also added.

**Review by mbercx — APPROVED** [review]
LGTM! 🚀

**Pull request merged**